plans/inhibit-if-kmods-is-not-supported/centos8/main.fmf

```diff
@@ -0,0 +1,14 @@
+discover:
+    how: fmf
+    # TODO filter inheritance doesn't work
```

Comment:
In order to [extend](https://fmf.readthedocs.io/en/stable/features.html#merging) values inherited from parent use the `+` suffix. Here's an example plan:

```yaml
discover:
    how: fmf
    filter:
      - tag:common-filter
execute:
    how: tmt

/centos8:
    discover+:
        filter+:
          - tag:centos8
/centos9:
    discover+:
        filter+:
          - tag:centos9
```

Which gives this:

```
> tmt plan show
/plans/example/centos8
    discover 
         how fmf
      filter tag:common-filter and tag:centos8

/plans/example/centos9
    discover 
         how fmf
      filter tag:common-filter and tag:centos9
```

Reply:
Thanks